.github/ISSUE_TEMPLATE/bug-issue.yml

```
@@ -105,16 +105,16 @@ body:
     validations:
       required: false
   - type: checkboxes
-    id: acknowledgements
+    id: acknowledgments

```

`acknowledgements` is fine.

```suggestion
    id: acknowledgements

```